tiledb/sm/config/config.cc

```diff
@@ -434,6 +436,8 @@ Status Config::unset(const std::string& param) {
     param_values_["sm.consolidation.mode"] = SM_CONSOLIDATION_MODE;
   } else if (param == "sm.vacuum.mode") {
     param_values_["sm.vacuum.mode"] = SM_VACUUM_MODE;
+  } else if (param == "sm.offsets_format") {
+    param_values_["sm.offsets_format"] = SM_OFFSETS_FORMAT;
```

Comment:
Misc general comments

Reply:
Good points, thanks! I have addressed them in the last version.